cpu/stm32_common/include/periph_cpu_common.h

```diff
@@ -43,10 +43,26 @@ extern "C" {
  * @brief   Available peripheral buses
  */
 enum {
```

Comment:
Now I think of it, there should be a guard anyway to check if RCC->APB2 exists for L0 or F0 since one/both of them only have APB1.